### ISSUE TITLE
Error in Console regarding bat-balance@1.0.0 Engine "node" Expected Version: 7.10.1

### ISSUE THREAD

**Issue by JMccormick264** [brave/browser-laptop#11339]
### Description

Node Version - Node 8.6.0
NPM: 5.3.0

When running "yarn install" I get an error in Console ```"error bat-balance@1.0.0: The engine "node" is incompatible with this module. Expected version "7.10.1"``` . Latest commit that I am on [Commit](https://github.com/brave/browser-laptop/commit/d972f89ce4fbae46862b5f08e00f32762cb45141)

### Steps to Reproduce

  1. Pull the latest commits
  2. Be on Node Version 8.6.0
  3. Run Yarn Install
  4. You should get the error that I linked above

**Actual result:**
Error Output:
```
yarn install v1.1.0-20171005.1705
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error bat-balance@1.0.0: The engine "node" is incompatible with this module. Expected version "7.10.1".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Expected result:**
I expect to be able to install all dependencies on Node 8.6.0
**Reproduces how often:** [What percentage of the time does it reproduce?]
100% of the time

### Brave Version
[Commit Here](https://github.com/brave/browser-laptop/commit/d972f89ce4fbae46862b5f08e00f32762cb45141)

Please open about:brave, copy the version information, and paste it.
 - Does not apply

**Reproducible on current live release:**
 - Does not apply

### Additional Information
 - N/A

**Comment by luixxiul** [comment]
I'm not sure if yarn install has been supported. https://github.com/brave/browser-laptop/issues/6437#issuecomment-271159238

**Comment by JMccormick264** [comment]
I am having issues with NPM. It says you can also use Yarn
`Instead of npm install you may also install with yarn.`

It worked before though. Not sure what went wrong now.

**Comment by JMccormick264** [comment]
Issue has been fixed commit [269c4f4](https://github.com/brave-intl/bat-publisher/commit/269c4f476d66ae39e0509c537424b1245f7986ad)
